src/aliases.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+$classMap = [
```

Comment:
I'm worried about this map falling out of date if we add new classes. Would be nice to test that all the classes in the `src` folder within the `Google\\` namespace have a mapping. Some combination of `RecursiveDirectoryIterator` and [`FileReflector`](https://github.com/phpDocumentor/Reflection/blob/3.0/src/phpDocumentor/Reflection/FileReflector.php) maybe.

Also, maybe just my preference, but I'd reverse the keys and values of this array. Makes more sense to me that the real name is the key, and also corresponds better to the argument order of `class_alias()`.

Reply:
> I'm worried about this map falling out of date if we add new classes

Stupid outsider question (apologies if this isn't appropriate here!): why would a _new_ class still be relevant for the old non-namespaced version? Would this still be maintained _in parallel_ … 🤔

Thanks!

Reply:
That's a good point, I suppose it's something that we'd need to consider. It would seem strange to have some classes without aliases, but maybe that's good, might be a nudge towards upgrading to use the namespace.

Reply:
I switched the order of `aliases.php` 

I agree with @mfn that there'd be no reason to add an alias for a new class